media-gfx/geeqie/geeqie-9999.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools xdg-utils git-r3
+
+DESCRIPTION="A lightweight GTK image viewer forked from GQview"
+HOMEPAGE="http://www.geeqie.org"
+EGIT_REPO_URI="https://github.com/BestImageViewer/geeqie"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ~ppc x86"
```

Comment:
do not commit straight to stable

Reply:
(Hint @grumpytetra: it's a live ebuild, they don't have keywords. That's why the CI bot is unsatisfied, too)

Reply:
yeah, I thought as much. I do see a fair number of other live ebuilds with keywords in the tree though.